app/models/installed_technology.rb

```diff
@@ -214,10 +214,18 @@ def as_json(*)
   end
 
   def yearly_investment
-    om_costs_per_year.to_f                   +
-    initial_investment.to_f                  / (technical_lifetime || 1) +
-    om_costs_per_full_load_hour.to_f         / (full_load_hours || 1) +
-    om_costs_for_ccs_per_full_load_hour.to_f / (full_load_hours || 1)
+    (initial_investment.to_f / (technical_lifetime || 1)) + yearly_om_cost
+  end
+
+  def yearly_om_cost
+    om_costs_per_year.to_f + (
+      om_costs_per_full_load_hour.to_f +
+      om_costs_for_ccs_per_full_load_hour.to_f
+    ) / yearly_full_load_hours
+  end
```

Comment:
@antw not really sure about this code formatting. What is prefered in these situations?